pkg/apply/applier.go

```diff
@@ -176,7 +182,12 @@ func (a *Applier) Run(ctx context.Context, invInfo inventory.InventoryInfo, obje
 			InventoryPolicy:        options.InventoryPolicy,
 		}
 		// Build list of apply validation filters.
-		applyFilters := []filter.ValidationFilter{}
+		applyFilters := []filter.ValidationFilter{
+			filter.InvalidObjectFilter{
+				InvalidIds: invalidObjects,
+				Policy:     options.ValidationPolicy,
+			},
+		}
 		if options.InventoryPolicy != inventory.AdoptAll {
 			applyFilters = append(applyFilters, filter.InventoryPolicyApplyFilter{
```

Comment:
Does the order of these filters matter?

Reply:
They get executed in this order. Filtering invalid objects first prevents the other filters from erroring due to an invalid object.